src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const handleClick = (tab) => {
+    if (tab.id !== selectedTab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  if (selectedTabId === undefined) {
+    onTabSelected(tabs[0].id);
+  }
```

Comment:
i think this is redundant logic, do we really have case when selectedTabId === undefined? if yes, let's check it when  we set tab